Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: setup test run lint clean
```

Comment:
since we are in the python ecosystem let's use a python project manager of task manager (like just or other open to suggestions)